docs/sdk/client-side-sdks/react-native.md

```diff
@@ -46,6 +71,229 @@ export default withDVCProvider(
 })(App)
 ```
 
+A complete working example of an `App.ts` file is below:
+```jsx
+import { View, Text } from "react-native";
+
+
+import 'react-native-get-random-values'
+import DeviceInfo from 'react-native-device-info'
+import { withDVCProvider } from '@devcycle/devcycle-react-sdk'
+
+global.DeviceInfo = DeviceInfo
+function App() {
+  return (
+    <View
+      style={{
+        flex: 1,
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
+      <Text>Universal React with Expo</Text>
+    </View>
+  );
+}
+
+export default withDVCProvider(
+{
+    envKey: 'client-xxxxxxxxxxxxxxxxxxx,
+    options: {
+        reactNative: true
+    }
+})(App)
+```
+
 ## Getting Started
 
-Please refer to the [DevCycle React SDK](/docs/sdk/client-side-sdks/react) documentation.
+There are two ways to initialize the SDK:
+* Non-Blocking: This loads your application and makes a request to initialize the SDK in the background. Once this request is complete,
+  your application will be ready to use the SDK.
+* Blocking: This allows you to delay the rendering of your application until the request to initialize the SDK is completed.
+
+To use these providers, you must grab the Environment Key from the DevCycle Dashboard.
+You can optionally pass in a user object to the provider to initialize the SDK.
+If you do not pass in a user to the provider, it will create an anonymous user and initialize the SDK with it.
+You can then call the `identifyUser` method on the client once the user has been authenticated.
+See [Identifying Users & Setting Properties](/docs/sdk/features/identify) for more details.
+
+### Non-Blocking
+
+The withDVCProvider function initializes the React SDK and wraps your root component. This provider may cause your app
+to flicker when it is first rendered, as it is waiting for the SDK to initialize.
+
+```js
+import { withDVCProvider } from '@devcycle/devcycle-react-sdk'
+```
+```js
+export default withDVCProvider({ envKey: 'ENV_KEY' })(App)
+```
+
+### Blocking
+
+The asyncWithDVCProvider function is similar to the withDVCProvider function, but allows you to block rendering of your application
+until SDK initialization is complete. This ensures your app does not flicker due to value changes.
+
+```js
+import { asyncWithDVCProvider } from '@devcycle/devcycle-react-sdk'
+```
+```js
+(async () => {
+    const DVCProvider = await asyncWithDVCProvider({ envKey: 'ENV_KEY' })
+
+    ReactDOM.render(
+        <DVCProvider>
+            <App />
+        </DVCProvider>
+    )
+})();
+```
+
+## Usage
+
+### Getting a Variable
+
+The SDK provides a hook to access your DevCycle variables:
+
+#### useVariableValue
```

Comment:
This one is a different sub header than `useDVCClient`

Reply:
this was deliberate because it's meant to be inside the "Getting a Variable" section